spec/services/create_remote_recipient_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe CreateRemoteRecipient do
+  describe '.call' do
```

Comment:
como o `call` é uma método de instância aqui fica `#call` se for um metodo de classe vc faz `.call` :)